didx-xyz/yoma-mobile
# YOMA-593 - feat(Challenge Credential Form): get form up to scratch

## Scope [YOMA-593](https://yomaworld.atlassian.net/browse/YOMA-593)
-first round on getting the Challenge Credential Form working correctly

![](https://media.giphy.com/media/VSe6e3019bJrG/giphy.gif)

## Work Done
- refactored the form somewhat to fit new criteria (select a pre-existing challenge only)
- adding real challenges to the form
- updated some of the existing form components
- adding status bar styling to the landing/auth screens and fixed up the color of the text throughout the app
- refactored/renamed some things. 

## Steps to Test
- make sure tests are running
- have a run through the add Challenge credential form.

## Screenshots
<img width="443" alt="Screenshot 2021-09-15 at 23 46 57" src="https://user-images.githubusercontent.com/1424561/133514233-d555b958-99e1-4205-806a-86cbddce8856.png">
<img width="431" alt="Screenshot 2021-09-15 at 23 47 09" src="https://user-images.githubusercontent.com/1424561/133514242-644f62a0-86ca-42a1-9eb7-ef5543e8ddbe.png">
